Index: try2/fetch.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package try2;\r\n\r\npublic class fetch {\r\n// pull deyince otomatik alir ve birlestirir from remote repositry \r\n  // ornegin burayi pull ile\r\n  // eger fetch ile cekersenn git fetch\r\n\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/try2/fetch.java b/try2/fetch.java
--- a/try2/fetch.java	(revision b035fd62fafcbd9440cece0ad8e7f21605ac5dc6)
+++ b/try2/fetch.java	(date 1657249648835)
@@ -5,4 +5,6 @@
   // ornegin burayi pull ile
   // eger fetch ile cekersenn git fetch
 
+    //5
+
 }
